opus/application/static_media/js/widgets.js

```diff
@@ -409,6 +409,27 @@ var o_widgets = {
                 o_hash.updateHash();
             }
 
+            // If we have a widget with qtype options any/all/only, we will append the (i) icon.
+            if ($(`#widget__${slug} select[name="${qtype}"]`).length !== 0) {
```

Comment:
What do you think about combining the previous 'if' block and just do all the "${qtype}" business together?